content/cpp/concepts/math-functions/terms/exp2/exp2.md

```diff
@@ -0,0 +1,84 @@
+---
+Title: 'exp2()'
+Description: 'Returns the base-2 exponential function, i.e. 2 raised to the given argument'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`exp2()`** is a builtin function in C++ STL that computes the `base-2` exponential function of a given number, i.e. `2` raised to the given argument.
+
+## Syntax
+
+```pseudo
+exp2(x)
+```
+
+It takes a single mandatory argument (can be positive, negative or 0).
+The `exp2()` function returns the value in the range of `[0, ∞]`.
+
+If no errors occur, the base-2 exponential of `n` is returned.
```

Comment:
These lines, in addition to the "Error Handling" lines should all be in a single bulleted list.

Reply:
Please check if the list is as expected.